test/test_api.py

```diff
@@ -69,9 +65,6 @@ def test_open_mf_dataset(self):
                         constants.NNODES_outCSne30)
         nt.assert_equal(len(uxds_mf_ne30.uxgrid._ds.data_vars),
                         constants.DATAVARS_outCSne30)
-        nt.assert_equal(uxds_mf_ne30.uxgrid.source_grid,
-                        str(self.gridfile_ne30))
-        nt.assert_equal(uxds_mf_ne30.source_datasets, self.dsfiles_mf_ne30)
```

Comment:
Why removing this?

Reply:
Same as above.

Reply:
The same, `uxds_var2_ne30.source_datasets` exists.

Reply:
Reintroduced